### PR TITLE
Wu Hong et al

### PR DESCRIPTION
**Pull request by inamoto4**
added information with regards to him and Xu Chenggang

### CODE REVIEW

File: data/Person.csv [hunk 1]
```diff
@@ -1,405 +1,405 @@
...
+P0023,杨,三白,zh,,,,,,,,,,,,,T00011,75,,,,,2017-07-05,LH,2017-11-17,MG
+P0024,Song,Haiquan,en,male,,,,,1936,,,,,,,T00011,75 79f,,,,,2017-07-05,LH,2017-11-17,MG
+P0024,宋,海泉,zh,male,,,,,1936,,,,,,,T00011,75 79f,,,,,2017-07-05,LH,2017-11-17,MG
+P0025,Gan,Tiesheng,en,male,,,,,1946,,,,,,,T00011,75-76  80,,,,,2017-07-05,LH,2017-11-17,MG
+P0025,甘,铁生,zh,male,,,,,1946,,,,,,,T00011,75-76  80,,,,,2017-07-05,LH,2017-11-17,MG
+P0026,Lu,Shuangqin,en,,,,,?,,,,,,,Yes,T00011,76f,,,,,2017-07-05,LH,2017-11-17,MG
+P0026,鲁,双芹,zh,,,,,?,,,,,,,Yes,T00011,76f,,,,,2017-07-05,LH,2017-11-17,MG
+P0027,Zhang,Xinhua,en,,,,,,,,,,,,,T00011,76 80,,,,,2017-07-05,LH,2017-11-17,MG
+P0027,张,新华,zh,,,,,,,,,,,,,T00011,76 80,,,,,2017-07-05,LH,2017-11-17,MG
+P0028,Zhang,Jiuxing,en,,,,,,,1972 or 1973,,,,,,T00011,76,,,,,2017-07-05,LH,2017-11-17,MG
```
Comment (duncdrum) [inline data/Person.csv]: Please use EDTF as in other similar cases see [wiki FAQ](https://github.com/LenaHenningsen/ReadingData/wiki/FAQ)

File: data/Place.csv [hunk 2]
```diff
@@ -81,7 +81,7 @@ PL0079,Saratow,51.592365,45.960804,2017-09-29,MG,2017-09-29,MG,
 PL0080,Shaoshan,27.905996376,112.474664768,2017-10-03,MG,2017-10-03,MG,
 PL0081,Huangbei,29.758889,118.534167,2017-10-03,MG,2017-10-03,MG,
 PL0082,Suzhou,31.30408,120.59538,2017-10-03,MG,2017-10-03,MG,
-PL0083,Landport,50.803,-1.087,2017-10-08,MG,2017-10-08,MG,
+PL0083,Landport,50.803000,-1.087000,2017-10-08,MG,2017-10-08,MG,
```
Comment (duncdrum) [inline data/Place.csv]: 👍

File: data/helper/data_dictionary.csv [hunk 3]
```diff
@@ -19,7 +19,7 @@ Person,source_id_1,Reference to text in which relevant information to an author
 Person,page_number_1,Page number in a text that the column source_id refers to; for several pages use 7-10. Add 'ONLINE SOURCE' for online sources,,2017-07-04,LH,2017-11-03,OS
 Person,source_id_2,2nd reference to text in which relevant information to an author is recorded. This can be an URL as well.,,2017-12-29,OS,2017-12-29,OS
 Person,page_number_2,Page number in a text that the column source_id refers to; for several pages use 7-10. Add 'ONLINE SOURCE' for online sources,,2017-12-29,OS,2017-12-29,OS
-Person,commentary,Please only add information relevant for our project (like '手抄本') or which helps to trace it (like 'part of xy anthology') or about its general importance. Keep it as short as possible and as long as necessary.,,2017-10-27,OS,2017-10-27,OS
+Person,commentary,"Please only add information relevant for our project, such as daugher of or friends with xy. Keep it as short as possible and as long as necessary.",,2017-10-27,OS,2017-10-27,OS
```
Comment (duncdrum) [inline data/helper/data_dictionary.csv]: Rephrase to avoid comma

**Comment by inamoto4** [comment]
Ok, I'll leave the information regarding social ties in the commentary column and transfer it as soon as we have settled on sth new. 

I have used the new format of [1972, 1973].

close #140

**Pull request merged**